src/main/java/io/github/coolmineman/cheaterdeleter/modules/packetcount/PacketLimiterCheck.java

```diff
@@ -21,6 +21,22 @@ public PacketLimiterCheck() {
     public ActionResult onPacket(CDPlayer player, Packet<ServerPlayPacketListener> packet) {
         if (!enabledFor(player)) return ActionResult.PASS;
         PacketVolumeData data = player.getOrCreateData(PacketVolumeData.class, PacketVolumeData::new);
+
+        if (packet.getClass().getName().equals("net.minecraft.class_2840")) {
```

Comment:
definitely a far better way to do that

Reply:
```java
if (packet instanceof CraftRequestC2SPacket) {
```
This is how this is usually done in networking